### PR TITLE
Add Select Parameter for Parameter Package

### PR DESCRIPTION
**Pull request by ismailsunni**

### CODE REVIEW

File: safe_extras/parameters/numeric_parameter.py [hunk 1]
```diff
@@ -9,9 +9,6 @@
 class NumericParameter(GenericParameter):
     """A subclass of generic parameter that accepts float or integer only.
 
-    Known issue : You should set the minimum and the maximum value explicitly.
-    See https://github.com/inasafe/inasafe/issues/2468#issuecomment-171124507
-
```
Comment (Gustry) [inline safe_extras/parameters/numeric_parameter.py]: Is-it no more a known issue ? Did you fix it ?

**Comment by Gustry** [comment]
Looks good

File: safe_extras/parameters/numeric_parameter.py (hunk 1 above)
Comment (ismailsunni) [inline safe_extras/parameters/numeric_parameter.py]: Hmm, I did nothing regarding this issue. I think I will need to check the status of parameter in inasafe and the parameter repo itself.
After looking deeper, there are some difference between the two which shouldn't happen.

Comment (Gustry) [inline safe_extras/parameters/numeric_parameter.py]: yes, I made some changes last year in inasafe/inasafe and i wasn't aware about inasafe/parameter repo.

Comment (Gustry) [inline safe_extras/parameters/numeric_parameter.py]: you don't want a GIT submodule ?

Comment (ismailsunni) [inline safe_extras/parameters/numeric_parameter.py]: There was a discussion about that, at the end we decided to not use git submodule or subtree. But, I can't remember what's the reason. Perhaps @timlinux remember about it?

Anyway, for short solution I will merge the update from InaSAFE repo to the parameter repository.

**Comment by Gustry** [comment]
Looks good again

**Pull request merged**